viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1321,14 +1325,22 @@ export class Cognite3DViewer {
     return true;
   }
 
-  private readonly startPointerEventListeners = () => {
-    const mouseHandler = new InputHandler(this.domElement);
+  /**
+   * Method for deleting all external events that are associated with current instance of Cognite3DViewer.
+   */
+  private disposeOfAllEventListeners() {
+    const cognite3DViewerEvents = ['click', 'hover', 'cameraChange', 'sceneRendered', 'disposed'];
```

Comment:
Is there anything we can do to make this list dynamic based on the events we actually support? E.g. just loop over all member of `this._events`?

Reply:
Or maybe we should introduce an `EventsHelper` class that can help us maintain this? I see that the code is duplicated to basically anywhere we have these listeners so might be worth creating an utility to avoid duplication.

Reply:
Yeah, we can just iterate through members of this._events, not sure why I decided not to do that 🥲. There are additional classes already present: EventTrigger and InputHandler. If we want to keep logic of `someClass.on/off('event', ...);` then we will have to extend each class to `EventsHelper`. If it's ok, I can create such class in another PR.

Reply:
I was thinking to have a `EventsHelper`-class that contained all individual events (e.g. 'click', 'hover', etc), but I realize now that it doesn't make sense since that would work well with types. Might be that you can create a generic implementation of `disposeOfAllEventListeners()` though, and just call this from each of the classes where you now have implemented it?

Not sure what your proposal about "have to extend each class to EventsHelper", but interested in hearing details if you have an idea.

Reply:
Added common method for all classes